onnxscript/function_libs/torch_lib/new_graph_prototype.py

```diff
@@ -0,0 +1,626 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+from __future__ import annotations
+
+from typing import Any, Dict, List, Mapping, Optional, Sequence, Tuple, Union
+
+import numpy as np
+import onnx
+import onnx.checker
+import onnx.defs
+import onnx.helper
+import onnx.shape_inference
+import torch
+from typing_extensions import TypeAlias
+
+import onnxscript
+from onnxscript import evaluator, onnx_opset
+from onnxscript import tensor as onnxscript_tensor
+from onnxscript._internal import param_manipulation, runtime_typing
+
+__all__ = [
+    "Graph",
+    "GraphEvaluator",
+    "GraphTensor",
+]
+
+
+ValidArgumentType: TypeAlias = Union[
```

Comment:
Maybe something like `Attribute` or `AttributeLike` would be a more meaningful name